### DISCUSSION TITLE
LazyVim 14 - fzf-lua on Windows

### DISCUSSION THREAD

**Discussion by centuryx476** [LazyVim/LazyVim#5121] [General]
Hello,
I am on Windows and I cannot for the life of me get Fzf to work. When I updated LazyVim to 14 half my config broke. I'm starting with Fzf to fix... 
Any help ?

![Screenshot 2024-12-14 203555](https://github.com/user-attachments/assets/2f42938f-3ce9-4f6c-99ce-9c01d27ab42d)

**Comment by folke** [comment]
You need to have `fzf` installed for `fzf-lua`, but you can also just
switch back to `telescope` and `fzf-lua` with `:LazyExtras` if you don't figure it out.

**Reply by centuryx476** [reply]
It's this one? : https://github.com/junegunn/fzf

**Reply by folke** [reply]
`scoop install fzf`

**Reply by centuryx476** [reply]
I don't use scoop

**Reply by centuryx476** [reply]
I think I got it

![image](https://github.com/user-attachments/assets/ae81b030-2520-4fbb-886e-e20c88f125e7)

**Reply by centuryx476** [reply]
I got past the first error message. Now this

And I am in a git repo
![image](https://github.com/user-attachments/assets/d6ba669e-3ef3-490a-b44f-8d138de98cb4)

**Reply by centuryx476** [reply]
Nvm, 
Just cleared out nvim-data

![image](https://github.com/user-attachments/assets/4970ae2c-a374-4cd8-9c3e-6f3c0783991e)

**Comment by aniketgm** [comment]
`winget install --id junegunn.fzf --exact --source winget`
